projects/js-packages/charts/src/types.ts

```diff
@@ -69,7 +70,7 @@ export type ChartTheme = {
 	/** Array of colors used for data visualization */
 	colors: string[];
 	/** Optional CSS styles for grid lines */
-	gridStyles?: CSSProperties;
+	gridStyles?: GridStyles;
```

Comment:
I noticed this is the same type for the `XYChartTheme` that is defined in Visx and thought we could use the same type for consistency? let me know thoughts - https://github.com/airbnb/visx/blob/86738178e86fdc0feacf83ac96e68724da0627c2/packages/visx-xychart/src/types/theme.ts#L38